docs/02-tutorial.mdx

```diff
@@ -27,22 +27,151 @@ We strongly recommend reading the linked sections of the documentation for more
+Refer to [the installation documentation](https://adoptium.net/installation) for all details.
+Arch-based Linux distributions can also install the Eclipse Temurin JDK via the associated non-official [AUR package](https://aur.archlinux.org/packages/jdk-temurin).
+
+#### sbt
+
+The PCE is developed with Scala.
+As this tutorial will require you to build the PCE from source, we'll use sbt, the de-facto build tool in the Scala community.
+
+The sbt [download](https://www.scala-sbt.org/download.html) page will give you instructions to install it:
+
+- on any system via [`cs setup`](https://www.scala-lang.org/download/), the Scala installer powered by Coursier
+- on Mac OS via SDKMAN or homebrew
+- on Windows via a msi file, Chocolatey or Scoop
+- on Linux via a deb or rpm package, or a universal package
+
+Refer to the more extensive [sbt "installing sbt" documentation](https://www.scala-sbt.org/1.x/docs/Setup.html) for more information.
+
+#### docker-compose
+
+In the following steps, we'll also use a docker-compose configuration to run the PCE with its required dependencies (including a PostgreSQL instance).
+
+You'll therefore either need [Docker Desktop](https://docs.docker.com/desktop/) (which provides a full Docker installation, including Compose) or the [Compose standalone](https://docs.docker.com/compose/install/other/) installation.
+
+:::caution Remember the Start the Docker Daemon
+
+Make sure the Docker daemon is running and accessible to your current user before anything.
+
+When using Systemd, you can run `sudo systemctl status docker` to check the status of the Docker daemon, and `sudo systemctl start docker` to start it.
+
+:::
+
+### Setup and Run the PCE
+
+Now, to clone the [privacy-computation-engine](https://github.com/blindnet-io/privacy-computation-engine/) repository, run:
+
+```bash
+git clone git@github.com:blindnet-io/privacy-computation-engine.git
+cd privacy-computation-engine
+```
+
+From the root directory of this repository, run the following `sbt` command to build the project and create a dedicated Docker image:
+
+```bash
+sbt docker:publishLocal
+```
+
+Then, the following script can be used to start a Postgres instance, execute database migrations and run the Privacy Computation Engine all at once using docker-compose:
+
+```bash
+./scripts/start.sh
+```
+
+After this script has been executed successfully, a new PCE instance should be available one your machine, with its API exposed on port `3000`.
+
+You can verify the service is running and available by calling the `health` endpoint:
+
+```bash
+curl -v localhost:9000/v0/health
+```
+
+:::caution Remember to stop it
+
+After completing this tutorial, make sure to stop and clean up all associated docker containers with:
+
+```bash
+./scripts/stop.sh
+```
+
+:::
```

Comment:
Too long for an introductory tutorial (ok for now, but should change soon)

I may address this by uploading prebuilt PCE images (ping @m4rk055, I don't remember if we wrote an issue about that)

Reply:
This one https://github.com/blindnet-io/devrel-management/issues/72?